tests/test_callbacks.py

```diff
@@ -36,6 +36,36 @@ def test_classification_eval_callback(self):
         eval_callback = callbacks.EvalCallBack(model, test_x, test_y, step=1)
         model.fit(train_x, train_y, callbacks=[eval_callback], epochs=1)
 
+    def test_classification_checkpoint_callback(self):
+        train_x, train_y = SMP2018ECDTCorpus.load_data()
+        test_x, test_y = SMP2018ECDTCorpus.load_data('test')
+
+        train_x = train_x[:1000]
+        train_y = train_y[:1000]
+        model = BLSTMModel()
+        model_path = os.path.join('model_saved', 'test_classification_checkpoint')
+        checkpoint = callbacks.KashgariModelCheckpoint(model_path,
+                                                       verbose=1,
+                                                       save_best_only=True,
+                                                       save_weights_only=False,
+                                                       kash_model=model)
+        model.fit(train_x, train_y, test_x, test_y, epochs=2, callbacks=[checkpoint])
+
+    def test_labeling_checkpoint_callback(self):
+        train_x, train_y = ChineseDailyNerCorpus.load_data()
+        test_x, test_y = ChineseDailyNerCorpus.load_data('test')
+
+        train_x = train_x[:1000]
+        train_y = train_y[:1000]
+        model = Labeling_BiLSTM_Model()
+        model_path = os.path.join('model_saved', 'test_labelling_checkpoint')
+        checkpoint = callbacks.KashgariModelCheckpoint(model_path,
```

Comment:
We need to check the saved model. Maybe check it by loading the saved model, and run the fit and eval function.

Reply:
load, predict and retrain test added